src/app/issues-viewer/card-view/card-view.component.ts

```diff
@@ -44,8 +46,8 @@ export class CardViewComponent implements OnInit, AfterViewInit, OnDestroy, Filt
       this.issues$ = this.issues.connect();
 
       // Emit event when issues change
-      this.issues$.subscribe((issues) => {
-        this.issueLength = issues.length;
+      this.issues$.subscribe(() => {
+        this.issueLength = this.issues.count;
```

Comment:
While this accomplishes the same thing as before, my opinion is that it is clearer with the previous implementation. 
To elaborate, the reasoning behind the subscription is clearer since we are using the emitted value.

Reply:
The previous implementation was problematic actually, `issues.length` was only maximally the page size, for e.g. a column of 23 issues and page size 20 would mean `this.issueLength = 20`

Reply:
I see. In that case an excellent fix!